.chloggen/update_role_arn_docs.yaml

```diff
@@ -0,0 +1,16 @@
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
```

Comment:
There's no need for a changelog entry for documentation changes. I'll add the label "Skip changelog", please remove this file from the PR.

Reply:
Fair to me ! Done updating the changes !!!!